### ISSUE TITLE
(when compiling) include/gamma-randr.h:24:10: fatal error: xcb/randr.h: No such file or directory

### ISSUE THREAD

**Issue by jerzybrzoska** [tronje/radshift#3]
Attempting to compile it I executed:
```
make && sudo make install
```
`make` gave me the following output:
```
gcc -Wall -Wextra -Wpedantic -Iinclude -Os -flto -c src/gamma-randr.c -o build/release/gamma-randr.o
In file included from src/gamma-randr.c:26:
include/gamma-randr.h:24:10: fatal error: xcb/randr.h: No such file or directory
   24 | #include <xcb/randr.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:60: build/release/gamma-randr.o] Error 1
```
My environment: Ubuntu Desktop 22.04LTS.
Could I be missing any dependencies?

**Comment by jerzybrzoska** [comment]
Update:
`libxcb-randr0-dev` turned out to be the dependency I was missing.

**Comment by tronje** [comment]
Never got around to closing this issue. Thanks for your fix in #4
